packages/office-addin-manifest/test/test.ts

```diff
@@ -522,20 +522,20 @@ describe("Unit Tests", function() {
         }
         assert.strictEqual(result, `You need to specify something to change in the manifest.`);
       });
-      it("should handle an invalid manifest file path", async function() {
-        // call  modify, specifying an invalid manifest path with a valid guid and displayName
-        const invalidManifest = path.normalize(`${manifestTestFolder}/foo/manifest.xml`);
-        const testGuid = uuid.v1();
-        const testDisplayName = "TestDisplayName";
-        let result;
-        try {
-          await manifestInfo.modifyManifestFile(invalidManifest, testGuid, testDisplayName);
-        } catch (err) {
-          result = err.message;
-        }
+      // it("should handle an invalid manifest file path", async function() {
```

Comment:
There's something you can add to the "it()" function so it skips the test and logs that it is being skipped instead of having to comment out all the lines. I don't remember it off the top of my head.

Reply:
Google for "mocha skip test" brings up the answer quickly:  `it.skip(`

https://danielkorn.io/post/skipping-tests-in-mochajs/